src/openvas.c

```diff
@@ -367,6 +375,14 @@ start_single_task_scan (void)
     g_message ("Could not initialize openvas SSL!");
 #endif
 
+  if (prefs_get ("debug_tls") != NULL && atoi (prefs_get ("debug_tls")) > 0)
+    {
+      g_warning ("TLS debug is enabled and should only be used with "
+                 "care, since they may reveal sensitive information.");
```

Comment:
```suggestion
                 "care, since it may reveal sensitive information in the scanner logs.");
```

Maybe it should be also noted that enabling TLS debug is making the logs huge (similar to log_whole_attack).